File: Models/ProductInfo.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;
using System.Linq;
using System.ComponentModel.DataAnnotations.Schema;



namespace Demo.Models
{
    
    public class ProductInfo
    {
        [Key]
        [Required]
        public int ProductId { get; set; }
        [Required]
        
        
        public string ProductName {get;set;}
        
        [Required]
        public string ProductDescription {get;set;}
        [Required]
        public int ProductRate{get;set;}
        [Required]
        public int GroupId {get;set;}
    

       
        
    }
}

File: ClientApp/app/ProductInfo.ts
export class ProductInfo {
    ProductId:number;
    ProductName: string;
    ProductDescription:string;
    ProductRate:number;
    GroupId:number;
    
    
    constructor(_productid:number,_productname:string,_productdescription:string,_productrate:number,_groupid:number)
    {
        this.ProductId=_productid;
        this.ProductName=_productname;
        this.ProductDescription=_productdescription;
        this.ProductRate=_productrate;
        this.GroupId=_groupid;
       

    }
}